index.js

```diff
@@ -60,13 +60,35 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function(event){
+    toggleColor(event.target)
+
+})
```

Comment:
Nicely done! Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`